src/main/java/net/floodlightcontroller/websocket/WebSocket.java

```diff
@@ -0,0 +1,92 @@
+package net.floodlightcontroller.websocket;
+
+import java.io.IOException;  
+import java.util.Iterator;
+import org.eclipse.jetty.websocket.api.Session;
+import org.eclipse.jetty.websocket.api.WebSocketAdapter;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.core.*;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import net.floodlightcontroller.websocket.WebSocketManager;
+
+/**
+ * WebSocket class that handles connections between client and server
+ *
+ * @author luke.sanderson@carleton.ca
+ */
+public class WebSocket extends WebSocketAdapter
+{
+
+	private static final Logger logger = LoggerFactory.getLogger(WebSocket.class); 	// Class logger
+	private static final boolean DEBUG = true; //Set to false to disable excessive logging
+	public WebSocketManager activeSessions = WebSocketManager.getInstance(); //Singleton architecture so we do not reference the wrong server
+	private Session currSess;
+
+	
+	
+    @Override
+    public void onWebSocketConnect(Session sess)
+    {
+        super.onWebSocketConnect(sess);
+        currSess = sess;
+        activeSessions.addActiveSession(currSess);
+       
+        try {
+			sess.getRemote().sendString("Connection established to websocket");
```

Comment:
Can you add a log statement for when a new web socket is connected?

Reply:
the method addActiveSession(Session session) creates a log statement for when it is connected so I felt it was redundant to have one here as well.